utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java

```diff
@@ -377,4 +381,16 @@ public static void updateEnvVarInDeploymentConfig(String dcName, String key, Str
         dc.getSpec().getTemplate().getSpec().getContainers().get(0).setEnv(vars);
         getInstance().updateDeploymentconfig(dc);
     }
+
+    public static boolean hasPodIssuesPullingImage(Pod pod) {
+        return pod.getStatus().getContainerStatuses().stream().anyMatch(status ->
+            status.getState().getWaiting() != null
+                && "ImagePullBackOff".equals(status.getState().getWaiting().getReason()));
+    }
+
+    public static boolean hasPodIssuesErrImagePull(Pod pod) {
```

Comment:
Do we need 2 for this? I'd personally expect that `hasPodIssuesPullingImage` would return true even if the pod is in `ErrImagePull`

Reply:
Probably no. We can merge them together.

Reply:
Let's merge methods and then merge this :tada: